src/pages/Overview/OverviewLists.tsx

```diff
@@ -0,0 +1,152 @@
+  const [isAddonShown, setIsAddonShown] = useState(false);
+  const [newTaskTag, setNewTaskTag] = useState('new');
+
+  const taskCompleteHandler = (key: string) => {
+    setTasks((prev: Task[]) => {
+      const newArr = [...prev];
+      const item = newArr.find((item) => item.task === key);
+      item!.completed = !item!.completed;
+
+      return newArr;
+    });
+  };
+  const newTaskHandler = (e: SyntheticEvent<HTMLInputElement>) => {
+    setNewTask(e.currentTarget.value);
+  };
+
+  const addNewTask = () => {
+    setTasks((prev: Task[]) => {
+      const task = {
+        task: newTask,
+        completed: false,
+        status: newTaskTag,
+      };
+      setNewTask('');
+      return [task, ...prev];
+    });
+    setNewTask('');
+  };
+
+  const selectAfter = (
+    <Select
+      defaultValue="new"
+      className={
+        isAddonShown
+          ? classes.overview_tasks_addon_show
+          : classes.overview_tasks_addon
+      }
+      onChange={(val: string) => setNewTaskTag(val)}
+    >
+      <Select.Option value="new">new</Select.Option>
+      <Select.Option value="urgent">urgent</Select.Option>
+      <Select.Option value="default">default</Select.Option>
+    </Select>
+  );
+
+  const tasksToShow = tasks.slice(0, 3);
+
+  return (
+    <div className={classes.ovewview_lists_container}>
+      <OverviewTickets />
+      <List
+        className={classes.overview_lists_list}
+        header={
+          <div>
+            <div className={classes.overview_flex}>
+              <h2> Tasks</h2>
+              <Button type="link">View all</Button>
+            </div>
+            <p>
+              <span className={classes.overview_lists_list_span}>Today </span>
+            </p>
+          </div>
+        }
+        bordered
+        dataSource={tasks}
+      >
+        <List.Item
+          onFocus={() => setIsAddonShown(true)}
+          onBlur={() => setIsAddonShown(false)}
+        >
+          <Input
+            type="text"
+            value={newTask}
+            placeholder="Create new task"
+            className={classes.overview_tasks_input}
+            onChange={newTaskHandler}
+            addonAfter={selectAfter}
+          />
+          <Button
+            icon={<AddTask />}
+            className={classes.overview_lists__add_task_btn}
+            onClick={addNewTask}
+          ></Button>
+        </List.Item>
+        {tasksToShow.map(
+          (
+            {
+              task,
+              completed,
+              status,
+            }: {
+              task: string;
+              completed: boolean;
+              status: string;
+            },
+            i: number
+          ) => {
+            let color = '#F0F1F7';
+            let fontColor = '';
+            if (status === 'urgent') {
+              color = '#FEC400';
+            }
+            if (status === 'new') {
+              color = '#29CC97';
+            }
+            if (color === '#F0F1F7') fontColor = '#9FA2B4';
```

Comment:
it looks like mess
maybe create a function `resolveColorByStatus`
then this part of code will be better

```suggestion
            const color = resolveColorByStatus(status);
            const fontColor = (color === '#F0F1F7') '#9FA2B4' : '';
```